lib/ActivityStreams.js

```diff
@@ -66,6 +66,11 @@ ActivityStreams.prototype = {
           this.send(am.actions.Response("RECENT_BOOKMARKS_RESPONSE", links), worker);
         });
         break;
+      case am.type("RECENT_LIKS_REQUEST"):
```

Comment:
Actually, this code is not being executed, because we aren't receiving messages yet from content